components/form/FAutocompleteInput.vue

```diff
@@ -364,6 +373,15 @@ function forceValidation() {
 
 watch(isValid, forceValidation);
 
+const FMenuRef = ref()
```

Comment:
We always avoid to define variable names in PascalCase (reserved for Class names and TS Types)
```suggestion
const menuRef = ref()
```

Reply:
You can type it I guess:
```suggestion
const menuRef = ref<InstanceType<typeof FMenu>>()
```